pkg/controller/servicebindingrequest/binder.go

```diff
@@ -477,6 +477,9 @@ func (b *Binder) update(objs *unstructured.UnstructuredList) ([]*unstructured.Un
 		log := b.logger.WithValues("Obj.Name", name, "Obj.Kind", obj.GetKind())
 		log.Debug("Inspecting object...")
 
+		sbrNamespacedName := types.NamespacedName{Namespace: b.sbr.GetNamespace(), Name: b.sbr.GetName()}
+		annotationChanged := SetSBRAnnotations(sbrNamespacedName, &obj)
+		log.Debug("anno changed")
```

Comment:
Would be good to have `annotation` instead of `anno`

Reply:
I will remove this debugging as it is not useful.